matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -433,6 +440,63 @@ private static double admittanceEndToEngineeringUnitsForLine(double transmission
         return shuntAdmittanceAtEnd * sBase / (nominalVoltageAtEnd * nominalVoltageAtEnd) - (1 - nominalVoltageAtOtherEnd / nominalVoltageAtEnd) * transmissionAdmittance;
     }
 
+    private static void createDcLines(MatpowerModel model, ContainersMapping containerMapping, Network network) {
+        for (MDcLine mDcLine : model.getDcLines()) {
+            String id = getId(HVDC_LINE_PREFIX, mDcLine.getFrom(), mDcLine.getTo());
+            String bus1Id = getId(BUS_PREFIX, mDcLine.getFrom());
+            String bus2Id = getId(BUS_PREFIX, mDcLine.getTo());
+            String voltageLevel1Id = containerMapping.getVoltageLevelId(mDcLine.getFrom());
+            String voltageLevel2Id = containerMapping.getVoltageLevelId(mDcLine.getTo());
+            VoltageLevel voltageLevel1 = network.getVoltageLevel(voltageLevel1Id);
+            VoltageLevel voltageLevel2 = network.getVoltageLevel(voltageLevel2Id);
+            boolean isInService = isInService(mDcLine);
+            String connectedBus1Id = isInService ? bus1Id : null;
+            String connectedBus2Id = isInService ? bus2Id : null;
+            String csId1 = getId(CONVERTER_STATION_1_PREFIX, mDcLine.getFrom(), mDcLine.getTo());
+            String csId2 = getId(CONVERTER_STATION_2_PREFIX, mDcLine.getFrom(), mDcLine.getTo());
+            double losses = mDcLine.getLoss0() + mDcLine.getLoss1() * mDcLine.getPf();
+            voltageLevel1.newVscConverterStation()
+                    .setId(csId1)
+                    .setBus(connectedBus1Id)
+                    .setConnectableBus(bus1Id)
+                    .setVoltageRegulatorOn(true)
+                    .setVoltageSetpoint(mDcLine.getVf() * voltageLevel1.getNominalV())
+                    .setLossFactor((float) computeLossFactor1(mDcLine.getPf(), losses * 0.5))
+                    .add();
+            voltageLevel2.newVscConverterStation()
+                    .setId(csId2)
+                    .setBus(connectedBus2Id)
+                    .setConnectableBus(bus2Id)
+                    .setVoltageRegulatorOn(true)
+                    .setVoltageSetpoint(mDcLine.getVt() * voltageLevel2.getNominalV())
+                    .setLossFactor((float) computeLossFactor2(mDcLine.getPf(), losses * 0.5, losses * 0.5))
+                    .add();
+            network.newHvdcLine()
+                    .setId(id)
+                    .setConverterStationId1(csId1)
+                    .setConverterStationId2(csId2)
+                    .setR(0)
```

Comment:
There is no such information in the Matpower model?

Reply:
No, in matpower DC lines are modeled as two equivalent dummy generators.

![image](https://github.com/powsybl/powsybl-core/assets/51124986/3513b475-6692-46f3-81fb-9716a983003b)